packages/app-builder-backend/src/platform/apps.ts

```diff
@@ -0,0 +1,123 @@
+import fetch from 'node-fetch'
+import config from '../config.json'
+import {
+  AppSummaryModelPagedResult,
+  CreateAppModel,
+  ProblemDetails,
+  AppDetailModel,
+  CreateAppRevisionModel,
+  ApproveModel,
+} from '@reapit/foundations-ts-definitions'
+import { getIdFromCreateHeaders } from '../utils/get-id-from-create-headers'
+
+const { platformApiUrl } = config
+
+const API_VERSION = 'latest'
+
+export const getDeveloperApps = async (developerId: string, accessToken: string) => {
+  const res = await fetch(`${platformApiUrl}/marketplace/apps?PageSize=100&developerId=${developerId}`, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+      'api-version': API_VERSION,
+    },
+  })
+  const data: AppSummaryModelPagedResult = await res.json()
+
+  return data.data
+}
+
+export const createMarketplaceAppRevision = async (
+  appId: string,
+  appRevision: CreateAppRevisionModel,
+  accessToken: string,
+) => {
+  const res = await fetch(`${platformApiUrl}/marketplace/apps/${appId}/revisions`, {
```

Comment:
The utils-common package exports fetchers that handle all the fetch logic including returning the location headers etc. No point repeating etc https://github.com/reapit/foundations/blob/master/packages/utils-common/src/fetcher/index.ts

Reply:
cheers for the pointer, using fetcher now!